include/deal.II/distributed/grid_tools.h

```diff
@@ -134,6 +134,68 @@ namespace parallel
     exchange_cell_data_to_ghosts (const MeshType &mesh,
                                   const std::function<boost::optional<DataType> (const typename MeshType::active_cell_iterator &)> &pack,
                                   const std::function<void (const typename MeshType::active_cell_iterator &, const DataType &)> &unpack);
+
+    /**
+     * A structure that allows the transfer of cell data of type @p T from one processor
+     * to another. It corresponds to a packed buffer that stores a vector of
+     * CellId and a vector of type @p T.
+     *
+     * This class facilitates the transfer by providing the save/load functions
+     * that are able to pack up the vector of CellId 's and the associated
+     * data of type @p T into a stream.
+     *
+     * Type @p T is assumed to be serializable by <code>boost::serialization</code> (for
+     * example <code>unsigned int</code> or <code>std::vector@<double@></code>).
+     */
+    template <int dim, typename T>
+    struct CellDataTransferBuffer
+    {
+      /**
+       * vector to store IDs of cells to be transfered.
```

Comment:
Capitalize `v` here. Same for the next function

Reply:
`Vector` creates a hyperlink to dealii's vector. I will write `Standard vector` if you don't mind

Reply:
Yes, that works. Or "A vector to store...", which is actually better grammar anyway.

Reply:
updated with `A`.